### ISSUE TITLE
Tweak design of GDPR banner

### ISSUE THREAD

**Issue by cjyabraham** [cncf/cncf.io#682]
Tweak design across all subsites with the following changes. Before performing, make sure Gareth doesn't have any additional changes he wants.

- [x] Make links WCAG compliant by turning black [as done here](https://github.com/cncf/tag-contributor-strategy/commit/0df9f569139ae2bca1efe3713bbbec7b0cbd91c8)
- [x] Raise up Cookie Policy and Storage Preferences links on European popup by 1px so that they're centered in space

**Comment by cjyabraham** [comment]
See [changeset](https://github.com/cncf/cartografos/commit/533848901e4c9d38d44b8307a0847fa542f2086e) needed to fix WCAG issues on the subsites.

**Comment by cjyabraham** [comment]
Deployed on the main site. We can leave such minor tweaks like this on the subsites for a rainy day.
